frameworks/SapientML/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sapientml
```

Comment:
Please install the framework through the setup.sh script. It allows people to specify versions, source, and so on.

Reply:
We have included framework installation through setup.sh script.